velox/vector/arrow/Bridge.cpp

```diff
@@ -39,11 +39,22 @@ static constexpr size_t kMaxBuffers{3};
 class VeloxToArrowBridgeHolder {
  public:
   VeloxToArrowBridgeHolder() {
-    for (size_t i = 0; i < kMaxBuffers; ++i) {
+    buffers_.resize(num_buffers_);
+    bufferPtrs_.resize(num_buffers_);
+    for (size_t i = 0; i < num_buffers_; ++i) {
       buffers_[i] = nullptr;
     }
   }
 
+  void resizeBuffers(size_t buffer_count) {
```

Comment:
I suppose you only need to this if bufferCount is in fact larger that numBuffer?

also, camel case :)

Reply:
Done